### PR TITLE
Quick start guide

### PR DESCRIPTION
**Pull request by farhan3**
Added a quick start guide for service, which can be referenced by the SDKs. Since you can't see what the README file looks like here, see https://github.com/farhan3/gp-common/tree/quick-start-guide 

Please provide feedback below.

Note: hold off merging until new release.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,98 @@
...
+
+![New Instance](images/new-instance.png "New Instance")
+
+![Credentials](images/creds.png "Credentials")
+
+**Note:** If the service instance is connected to an app on Bluemix, the credentials can be accessed from the app's environment variables - the SDKs automatically take care of this.
+
+**Caution!** The default credentials shown above provide complete access to the service instance, including the ability to delete and modify bundles. When you are using the service in a production app, it is recommended that you create a new **Reader** account through the **API Users** tab. The Reader account can only read bundle data.
+
+<!-- the download anchor is required for backwards compatibility  -->
```
Comment (srl295) [inline README.md]: good. this was the first thing i was going to complain about…

File: README.md [hunk 2]
```diff
@@ -1,17 +1,98 @@
...
-[IBM Globalization Pipeline](https://www.ng.bluemix.net/docs/#services/GlobalizationPipeline/index.html) SDK projects.
+<p align="center">
+  <img src="images/icon.png"/>
+</p>
 
-![Globalization Pipeline Logo](images/catalog_icon.png "Globalization Pipeline")
 
+IBM Globalization Pipeline is a DevOps integrated application translation management service that you can use to rapidly translate and release cloud and mobile applications to your global customers. Access IBM Globalization Pipeline capabilities through its dashboard, RESTful API, or integrate it seamlessly into your application's Delivery Pipeline.
 
-DOWNLOAD
```
Comment (srl295) [inline README.md:11]: Should we keep the SDKs list at the top?

**Comment by srl295** [comment]
LGTM with comments but let's discuss.

**Comment by farhan3** [comment]
TODO: The Bluemix Catalog image under _Quick Start Guide_ will need to be updated since the beta entry will be removed on the same day as the new release.

**Comment by srl295** [comment]
merging… will need some follow up work.

**Pull request merged**